<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KVQuerySetterForm</class>
 <widget class="QWidget" name="KVQuerySetterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>53</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>53</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="3">
    <widget class="QComboBox" name="opCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QTableWidget" name="filtTable">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Keys</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Operations</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Values</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="keyCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QLineEdit" name="valueEdit">
     <property name="placeholderText">
      <string>Press Enter to Add</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" rowspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>âœ…</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
